layouts/partials/head.html

```diff
@@ -8,6 +8,10 @@
         <meta http-equiv="refresh" content="0; url={{ .Params.redirect_to }}" />
     {{ end }}
 
+    {{ if .Params.noindex }}
+        <meta name="robots" content="noindex">
```

Comment:
So, I've already used `noindex` elsewhere to mean "exclude this page in our (lame) Pulumi search index". This is repurposing `noindex` to mean `<meta name="robots" content="noindex">`.

I'd be fine with renaming existing uses of `noindex` to something that better explains what it means, e.g. `exclude_from_pulumi_search_index`, and repurposing `noindex` for `<meta name="robots" content="noindex">`.

Reply:
Ah, that is good to know. I renamed both of the variables to be more specific and added descriptions on how/when to use them in the contributing doc.